src/common/techstack/TechStackInfo.js

```diff
@@ -0,0 +1,96 @@
+import {
+  SiReact,
+  SiRedux,
+  SiGit,
+  SiMaterialui,
+  SiTailwindcss,
+  SiJavascript,
+  SiFirebase,
+  SiTypescript,
+  SiGraphql,
+  SiDocker,
+  SiHasura,
+  SiPostgresql
+} from "react-icons/si";
+import NHost from "images/nhost.svg";
+
+export const TechStackInfo = [
+  {
+    comp: SiReact,
+    text: "ReactJS",
+    type: "icon",
+    link: "https://reactjs.org",
+  },
+  {
+    comp: SiRedux,
+    text: "Redux",
+    type: "icon",
+    link: "https://redux.js.org",
+  },
+  {
+    comp: NHost,
+    text: "NHost",
+    type: "image",
+    link: "https://nhost.io",
+  },
+  {
+    comp: SiGit,
+    text: "Git",
+    type: "icon",
+    link: "https://git-scm.com",
+  },
+  {
+    comp: SiMaterialui,
+    text: "Material UI",
+    type: "icon",
+    link: "https://mui.com",
+  },
+  {
+    comp: SiTailwindcss,
+    text: "Tailwind",
+    type: "icon",
+    link: "https://tailwindcss.com",
+  },
+  {
+    comp: SiHasura,
+    text: "Hasura",
+    type: "icon",
+    link: "https://hasura.io",
+  },
+  {
+    comp: SiPostgresql,
+    text: "PostgreSQL",
+    type: "icon",
+    link: "https://www.postgresql.org",
+  },
+  {
+    comp: SiJavascript,
+    text: "Javascript",
+    type: "icon",
+    link: "https://www.ecma-international.org/",
+  },
+  {
+    comp: SiFirebase,
+    text: "Firebase",
+    type: "icon",
+    link: "https://firebase.google.com",
+  },
+  {
+    comp: SiTypescript,
+    text: "Typescript",
+    type: "icon",
+    link: "https://www.typescriptlang.org",
+  },
+  {
+    comp: SiGraphql,
+    text: "Graphql",
+    type: "icon",
+    link: "https://graphql.org",
+  },
+  {
+    comp: SiDocker,
+    text: "Docker",
+    type: "icon",
+    link: "https://www.docker.com",
+  },
```

Comment:
time to bring python,node etc.

Reply:
Yeah we should. Do we use plop anymore after your PR @koustov ? If not make sure to remove from package.json on your branch.

@Angryman18 this change is on you

Reply:
added